docs/in-depth.md

```diff
@@ -0,0 +1,7 @@
+In-Depth
+========
+
+Learn more about:
+
+ - [Firmware updates](in-depth/firmware-updates.md)
+ - [Using Decky Loader](in-depth/decky-loader.md)
```

Comment:
Hm... yeah... not written yet (and the latter has that PR that is not merged either.)